### PR TITLE
Update inshellisense commands and options

### PR DESCRIPTION
**Pull request by naquiroz**
# Description
This pull request solves #194 

# How is it solved

A new flag is added: `parent-term-exit` which allows users to opt in on closing any parent shells when `is` is started. This is useful for adding is to a `~/.zshrc` configuration.

### CODE REVIEW

**Comment by naquiroz** [comment]
@microsoft-github-policy-service agree

File: README.md [hunk 1]
```diff
@@ -22,11 +22,13 @@ After completing the installation, you can run `is` to start the autocomplete se
...
-| ------------------------------------- | ------- |
-| Start                                 | `is`    |
-| Stop                                  | `exit`  |
-| Check If Inside Inshellisense Session | `is -c` |
+| Action                                | Command                           | Description                                      |
+| ------------------------------------- | --------------------------------- | ------------------------------------------------ |
+| Start                                 | `is`                              | Start inshellisense session on the current shell |
+| Stop                                  | `exit`                            | Stop inshellisense session on the current shell  |
+| Check If Inside Inshellisense Session | `is -c`                           | Check if shell inside inshellisense session      |
+| Exit parent terminal                  | `is -p` or `is --parent-term-exit`| Exit parent terminal on exit, if exists. Useful for adding is to shell startup scripts like `~/.bashrc` or `~/.zshrc` |
```
Comment (cpendery) [inline README.md:30]: Let's not add this to readme, like `-V`, since it's a custom option that not many users will need.

File: src/index.ts [hunk 2]
```diff
@@ -27,6 +27,7 @@ program
   .action(action(program))
   .option("-s, --shell <shell>", `shell to use for command execution, supported shells: ${supportedShells}`)
   .option("-c, --check", `check if shell is in an inshellisense session`)
+  .option("-p, --parent-term-exit", `if the terminal with inshellisense is closed, exit the parent terminal`)
```
Comment (cpendery) [inline src/index.ts:30]:
```suggestion
  .option("--parent-term-exit", `when inshellisense is closed, kill the parent process`)
```
Can we update it to not have a short flag and remove anything about killing the parent terminal (a user could have a parent shell or some other parent process)

Comment (naquiroz) [reply src/index.ts]: Removed!

**Review by cpendery — APPROVED** [review]

**Comment by naquiroz** [comment]
Thats great thanks!

**Pull request merged**